src/config/getStatusBarHeight.js

```diff
@@ -0,0 +1,11 @@
+import { getStatusBarHeight as ovrGetStatusBarHeight } from 'react-native-status-bar-height';
+
+/* eslint-disable no-underscore-dangle */
+export const expoRoot = global.Expo || global.__expo || global.__exponent;
+export const expoManifest =
+  !expoRoot || !expoRoot.Constants || !expoRoot.Constants.manifest
+    ? {}
+    : expoRoot.Constants.manifest;
+
+export const getStatusBarHeight = (skipAndroid = false) =>
+  ovrGetStatusBarHeight(skipAndroid || !!expoManifest.androidStatusBar);
```

Comment:
this looks useful. would it make sense to submit this PR into `react-native-status-bar-height` directly so that we they can take advance of this as well?

Reply:
Yep. Calling from anywhere must take into account that expo could intercept StatusBar control

Reply:
@oxyii did you ever make that PR to them?

Reply:
@ovr will you merge this PR into `react-native-status-bar-height`?

Reply:
@oxyii Can you try with latest version `v2.4.0` and code `getStatusBarHeight (isExpo())`

refs https://github.com/ovr/react-native-status-bar-height/commit/7cae2c6f7cfb3227b91868c0580f601ce03150ca

Thanks!

Reply:
@ovr `skipAndroid` arg is false by default. Will you change it to `true` by default if expo and `androidStatusBar` exists in app.json. This is really useful I think

Reply:
> Will you change it to true by default if expo

It's require major bump

Why do you need to take a look `androidStatusBar`? As I understand you need a  "hack" for Expo, try to use `isExpo()` from `v2.4.0`

Reply:
@ovr can you re-check that expo handled StatusBar in all cases? My experiments shows that only `androidStatusBar` prop in app.json initiate to handle

Reply:
Is there any update on this?

Reply:
@iRoachie I see no one change on this inside `react-native-status-bar-height`